### DISCUSSION TITLE
gopls language support

### DISCUSSION THREAD

**Discussion by laplantePierre** [zed-industries/zed#14109] [Feature Requests]
In neovim, when I have an error it display the error as in:
<img width="466" alt="Capture d’écran, le 2024-07-10 à 17 27 04" src="https://github.com/zed-industries/zed/assets/145718980/09b97f62-df62-44f9-91cc-43bebd8b6c15">
In zed, this does not seems to be supported on my setting are incorrect:

{
  "theme": "Atelier Plateau Light",
  "vim_mode": true,
  "ui_font_size": 16,
  "buffer_font_size": 16,
  "cursor_blink": false,
  "languages": {
    "HTML": {
      "formatter": null,
      "format_on_save": false
    }
  },
  "lsp": {
    "gopls": {
      // "binary": {
      //   "path": "/Users/thorstenball/bin/gopls",
      //   "arguments": ["-debug=0.0.0.0:9090"]
      // },
      "initialization_options": {
        "usePlaceholders": true,
        "buildFlags": ["-tags=debug"], // <--------------- buildFlags!
        "completeUnimported": true,
        "experimentalPostfixCompletions": true,
        "hints": {
          "assignVariableTypes": true,
          "compositeLiteralFields": true,
          "compositeLiteralTypes": true,
          "constantValues": true,
          "functionTypeParameters": true,
          "parameterNames": true,
          "rangeVariableTypes": true,
          "importSuggestions": true,
          "packageHints": true,
          "typeInference": true
        }
      }
    }
  }
}

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
